### ISSUE TITLE
expo-image-picker doesn't show on top of modal on iOS

### ISSUE THREAD

**Issue by MerryCello** [expo/expo#13013]
### Summary

I have a modal (through react-native-raw-bottom-sheet) that has a button to change an image using expo-image-picker. The image picker on iOS (at least on iPhone 7 with version 14.5.1) will appear behind the modal. Closing out of the modal also closes out of the image picker.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

iOS

### SDK Version (managed workflow only)

40.0.0

### Environment

  Expo CLI 4.3.2 environment info:
    System:
      OS: Windows 10 10.0.19042
    Binaries:
      Node: 14.16.0 - C:\path\to\nodejs\node.EXE
      npm: 7.7.6 - C:\path\to\nodejs\npm.CMD
    npmPackages:
      expo: ~40.0.0 => 40.0.1
      react: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-40.0.0.tar.gz => 0.63.2
    Expo Workflow: managed

### Reproducible demo or steps to reproduce from a blank project

1. Install `react-native-raw-bottom-sheet`
2. Install `expo-image-picker`
3. Paste code in App.js:
``` Javascript
import { StatusBar } from 'expo-status-bar';
import React, {useEffect, useState} from 'react';
import {Button, StyleSheet, View} from 'react-native';
import RBSheet from "react-native-raw-bottom-sheet";
import * as ImagePicker from 'expo-image-picker';

export default App = () => {
  const [RbSheet, setRBSheet] = useState(null)
  const [RBSheetIsOpen, setRBSheetIsOpen] = useState(false)

  const selectImage = async () => {
    const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
    if (status === 'granted') {
      ImagePicker.launchImageLibraryAsync({
        mediaTypes: ImagePicker.MediaTypeOptions.Images,
        allowsEditing: true,
        aspect: [1, 1],
        quality: 1,
      });
    }
  }

  useEffect(() => {
    if (RbSheet && !RBSheetIsOpen) {
      RbSheet.open()
      setRBSheetIsOpen(true)
    }
  }, [RbSheet])

  return (
    <View style={styles.container}>
      <Button style={{flex: 1}} title={"Open RBSheet"} onPress={() => {
        RbSheet.open();
        setRBSheetIsOpen(true);
      }
      }/>
      <RBSheet
        ref={ref => { setRBSheet(ref); }}
        closeOnDragDown={true}
        onClose={() => setRBSheetIsOpen(false)}
      >
        <Button
          title={"Select an image"}
          onPress={selectImage}
        />
      </RBSheet>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
```

**Comment by cruzach** [comment]
Hi! This was fixed in SDK 41, you'll need to download the new version of Expo Go (2.19.6)

**Comment by cruzach** [comment]
Hi!

Thanks for the report, this was fixed in the latest version of Expo Go (2.19.6), and has been deployed to our builders as well (for SDK 41) so you can get the fix with `expo build:ios`

**Comment by MerryCello** [comment]
Oh sorry about that. I didn’t find any solution on this, so I submitted this issue. I was able to update the project to have SDK 41 and it did resolve the issue. Thank you   From: Charlie CruzanSent: Sunday, May 23, 2021 06:55To: expo/expoCc: Kevin Foniciello; AuthorSubject: Re: [expo/expo] expo-image-picker doesn't show on top of modal on iOS (#13013) Hi! This was fixed in SDK 41, you'll need to download the new version of Expo Go (2.19.6)—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
